README.md

```diff
@@ -1,2 +1,26 @@
-# raiden-services
-Development of the Raiden Monitoring and Pathfinding Services
+# Raiden Services
+
+Supplemental services for the [Raiden Network](https://raiden.network/).
+
+[![Build Status](https://travis-ci.com/raiden-network/raiden-services.svg?branch=master)](https://travis-ci.com/raiden-network/raiden-services)
+[![Coverage](https://img.shields.io/codecov/c/github/raiden-network/raiden-services.svg?style=round)](https://codecov.io/gh/raiden-network/raiden-services/)
+
+### Monitoring Service
+
+The Monitoring Service watches open payment channels when the user is not online. In case one of the user’s channel partners wants to close a channel while the user is offline, the monitoring service sends the latest balance proof to the channel smart contract and thus ensures the correct settlement of the channel.
+
+### Pathfinding Service
+
+The Pathfinding service supports users in finding the cheapest or shortest way to route a payment through the network. A pathfinding service relies on data from the network, the respective smart contract as well as information provided voluntarily by mediating nodes. This information consists of the mediation fees charged and the respective available channel capacities.
+
+## Getting started
+
+The Raiden Services require Python 3.7+.
+
+To install the Raiden services run the following commands:
+
+```sh
+virtualenv venv
```

Comment:
We could add `-p python3.7` here, for the case that the user's default python is older.